src/makeCalculator.js

```diff
@@ -3,8 +3,44 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(operate) {
+  const obj = {
+    result: 0,
+
+    reset: function () {
+      this.result = 0;
+
+      return this;
```

Comment:
Use the method shorthand syntax for defining methods in an object. Replace 'reset: function () {' with 'reset() {'.